src/microbe_stage/MicrobeStage.cs

```diff
@@ -786,6 +786,8 @@ protected override void SetupStage()
 
     protected override void OnGameStarted()
     {
+        tutorialGUI.CurrentLifeOrigin = GameWorld.WorldSettings.Origin;
```

Comment:
I'd just roll the current world settings into the event that triggers the new game tutorial anyway.

Reply:
Put it in _Ready()?